core/tests/unit/kmnkbd/test_actions_normalize.cpp

```diff
@@ -440,6 +440,23 @@ void run_actions_normalize_tests() {
     /* app_context: */                   u"a\U0001F607bca\U0001F60E"
   );
 
+  km_core_context_item items_11067[] = {
+    { KM_CORE_CT_CHAR,   {0,}, { U'𐒻' } },
+    { KM_CORE_CT_CHAR,   {0,}, { U'𐒷' } },
+    KM_CORE_CONTEXT_ITEM_END
+  };
+
+  // regression #11067
+  test_actions_normalize(
+    "A non-BMP char in context (#11067)",
+    /* app context pre transform: */     u"𐒻",
+    /* cached context post transform: */ u"𐒻𐒷",
+    /* cached context post transform: */ &items_11067[0],
+    /* action del, output: */            0, U"𐒻𐒷",
+    // ---- results ----
+    /* action del, output: */            1, U"𐒻𐒷",
+    /* app_context: */                   u"𐒻𐒷"
+  );
```

Comment:
Fails w/o this fix with:

```
test_actions_normalize: (A non-BMP char in context (#11067)): delete: 2 output: |𐒻𐒷|
U+104bb U+104b7
stderr:
Test failed with 0 at ../../../tests/unit/kmnkbd/test_actions_normalize.cpp:131:
  expected_delete == test_actions.code_points_to_delete
```

showing that it tried to delete 2 code points instead of 1